drivers/bridge/bridge.go

```diff
@@ -775,8 +777,8 @@ func (d *driver) DeleteNetwork(nid string) error {
 		return err
 	}
 
-	// We only delete the bridge when it's not the default bridge. This is keep the backward compatible behavior.
-	if !config.DefaultBridge {
+	// We only delete the bridge if it was created by the bridge driver.
+	if config.ifaceManaged {
```

Comment:
Based on my comment above, I think we should change this check to 

`if config.ifaceManaged && !config.DefaultBridge {`

And update the new comment accordingly

`// We only delete the bridge if it was created by the bridge driver and it is not the default one (to keep the backward compatible behavior.)`